src/CAREUI/misc/PaginatedList.tsx

```diff
@@ -33,7 +33,10 @@ function useContextualized<TItem>() {
 interface Props<TItem> extends QueryOptions<PaginatedResponse<TItem>> {
   route: QueryRoute<PaginatedResponse<TItem>>;
   perPage?: number;
-  children: (ctx: PaginatedListContext<TItem>) => JSX.Element | JSX.Element[];
+  children: (
+    ctx: PaginatedListContext<TItem>,
+    refetch: () => void
```

Comment:
@konavivekramakrishna how about passing the entire `query` object itself, so that later on, if we want to pass some other thing from the query, we won't have to keep adding positional arguments to it.

```suggestion
    query: ...
```

Reply:
got it
Thank you @rithviknishad